scripts/transpile.js

```diff
@@ -0,0 +1,11 @@
+const { transpileFiles } = require('@asyncapi/generator-react-sdk');
+const path = require('path')
+
+async function transpileTemplate() {
+  const templateContentDir = path.join(__dirname, '../template');
+  console.log("templateContentDir",templateContentDir)
+  const outputDir = path.join(__dirname, '../__transpiled');
```

Comment:
console log this variable as well, but again, with full descriptive console log info